packages/rfq-loadtest/config-template.yaml

```
@@ -0,0 +1,69 @@
+# if you only fund ETH on one of the test chains it will auto-rebalance the funds to the others before beginning tests.
+
+# Startup example w/ pyRepeater to auto-restart the process if anything kills it
+# python3 pyRepeater.py "node index.js --configFile ../config-prod.yaml --pKeyIndex 1"
+
+PRIVATE_KEY_1: "0xabcdef123456abcdef123456abcdef123456abcdef123456abcdef123456abcd"
+PRIVATE_KEY_2: "0xabcdef123456abcdef123456abcdef123456abcdef123456abcdef123456abcd"
+PRIVATE_KEY_3: "0xabcdef123456abcdef123456abcdef123456abcdef123456abcdef123456abcd"
+PRIVATE_KEY_4: "0xabcdef123456abcdef123456abcdef123456abcdef123456abcdef123456abcd"
+PRIVATE_KEY_5: "0xabcdef123456abcdef123456abcdef123456abcdef123456abcdef123456abcd"
+
+# these settings are a pace of about 100K deposits per day 
```

yamllint

[error] trailing spaces

(trailing-spaces)